src/scriptworker/constants.py

```diff
@@ -116,43 +116,14 @@
             {
                 "by-cot-product": immutabledict(
                     {
-                        "firefox": (
-                            "aws-provisioner-v1/gecko-1-decision",
-                            "aws-provisioner-v1/gecko-2-decision",
-                            "aws-provisioner-v1/gecko-3-decision",
-                            "gecko-1/decision",
-                            "gecko-2/decision",
-                            "gecko-3/decision",
-                        ),
-                        "thunderbird": (
-                            "aws-provisioner-v1/gecko-1-decision",
-                            "aws-provisioner-v1/gecko-2-decision",
-                            "aws-provisioner-v1/gecko-3-decision",
-                            "comm-1/decision",
-                            "comm-2/decision",
-                            "comm-3/decision",
-                        ),
-                        "mobile": (
-                            # gecko-focus was for mozilla-mobile releases (bug 1455290) for more details.
-                            # TODO: Remove it once not used anymore
-                            "aws-provisioner-v1/gecko-focus",
-                            "aws-provisioner-v1/mobile-1-decision",
-                            # We haven't had the need for mobile-2-decision yet
-                            # https://bugzilla.mozilla.org/show_bug.cgi?id=1512631#c6
-                            "aws-provisioner-v1/mobile-3-decision",
-                            "mobile-1/decision",
-                            "mobile-3/decision",
-                        ),
-                        "mpd001": ("aws-provisioner-v1/mpd001-1-decision", "aws-provisioner-v1/mpd001-3-decision", "mpd001-1/decision", "mpd001-3/decision"),
-                        "application-services": (
-                            "aws-provisioner-v1/app-services-1-decision",
-                            "aws-provisioner-v1/app-services-3-decision",
-                            "app-services-1/decision",
-                            "app-services-3/decision",
-                        ),
+                        "firefox": ("gecko-1/decision", "gecko-2/decision", "gecko-3/decision",),
+                        "thunderbird": ("comm-1/decision", "comm-2/decision", "comm-3/decision",),
+                        "mobile": ("mobile-1/decision", "mobile-3/decision",),
+                        "mpd001": ("mpd001-1/decision", "mpd001-3/decision"),
+                        "application-services": ("app-services-1/decision", "app-services-3/decision",),
                         "xpi": ("xpi-1/decision", "xpi-3/decision"),
-                        # TODO: Add adhoc-3 whenever we're ready to go to prod
-                        "adhoc": ("adhoc-1/decision",),
+                        "adhoc": ("adhoc-1/decision", "adhoc-3/decision"),
+                        "scriptworker-scripts": ("swscripts-1/decision", "swscripts-3/decision",),
```

Comment:
I think I would use `scriptworker` both for the CoT product, and the trust domain (which is the worker prefix).

Although, thinking ahead, I wonder if we can use one CoT product + trust domain for all the releng projects which are deployed by cloudoups? If we go this route, I'd be inclined to use `releng`, and migrate the few projects using that that we don't deploy to another trust domain.

Reply:
Hm. I think we may have technical issues -- solvable, but not necessarily without some work -- that keep us from using `releng` if, say, the trusted repos are in separate Github organizations. I do think we should be able to reuse the same pools across both the `scriptworker` and `scriptworker-scripts` repos, though, so I think going with `scriptworker` makes a lot of sense.

Reply:
Done. I also added the `project:scriptworker:pypi:production` restricted scope, in case we decide to do automatic pypi pushes from `scriptworker{,-scripts}`. These may change in the future.